skimage/segmentation/_watershed.py

```diff
@@ -35,7 +35,7 @@ def _validate_inputs(image, markers, mask, connectivity):
     markers : int or array of int
         The marker image.
     mask : array, or None
-        A boolean mask, True where we want to compute the watershed.
+        Array of same shape as image. Defaults to ndarray of 1s. May be used to restrict labeling to thresholded regions (see Examples section).
```

Comment:
There is no Examples section in this docstring...

I admit that I don't find this new parameter description any clearer/better than the current one. Also, "defaults to ..." and "may be used to ..." don't belong in the docstring of this private function:

* if you look at this function's signature, you'll see no default value for `mask`; iirc the default value is passed in the main function (`watershed`) which calls this function (`_validate_inputs`);
* advice/guidance such as "may be used" should be user-facing; it does not make sense in the documentation of a private function.